### ISSUE TITLE
Bugged logs data with inconsistent JSON format

### ISSUE THREAD

**Issue by HiIamJeff** [vck3000/ProAvalon#386]
**Describe the bug**
Hi there, I like to use the data to do some analysis, however, I found the Anonymized logs data (https://www.proavalon.com/gameRecordsData) from the Statistics page has inconsistent JSON format. 

**Detail**
While loading JSON file into Python with this 
```
with open('gameRecordsDataAnon.json') as infile:
    data = json.load(infile)
```
There would be an error message:
```
error message: JSONDecodeError: Extra data: line 1 column 197353 (char 197352)
```

Looking into the string, I found there are several errors that should be a comma instead of '**][**', so the program couldn't load the JSON file correctly.
Please see the following example:
```
"whoAssassinShot":"Merlin","__v":0}][{"spyTeam":["c","f"],"resistanceTeam":["a","b","d","e"],
```
Please use these codes to see more example:
```
json_data = open("gameRecordsDataAnon.json").readlines()
json_data[0][194000:202400] # please find the gap between different records
```

Thanks!

Jeff

**Comment by HiIamJeff** [comment]
Hi,

I did a very simple clean for the JSON file as below and made it works. I think there might be some problems when integrating the records from different batches together. 

Another thing I like to ask is if you can provide up-to-date game data on the website. The one provided currently only had the data until 2020-01. I am looking forward to using this data for an analysis of game patterns of Avalon as a side project (just for practicing and not for profit). Hope this won't be too much effort. Thanks!

Jeff

```
import json
import pandas as pd
from pprint import pprint

### read the file
with open('gameRecordsDataAnon.json', 'r') as f:
    json_data = f.readlines()

## fix the json
json_data = json_data[0].replace('][', ',') # fix
json_data = json.loads(json_data)

print('complete cleaning!')
print('==========')
pprint(json_data[0])
```

**Comment by vck3000** [comment]
Hey Jeff, feel free to contact me via discord or email so we can chat more. My details are on [here](https://www.proavalon.com/about).

**Comment by vck3000** [comment]
As discussed in our chat, the problem stems from [here](https://github.com/vck3000/ProAvalon/blob/master/routes/index.js#L487). 

Instead of JSON.stringifying every "block", the stringify should happen at the end with the whole records data. However, the whole records data is too large to store in memory (~600MB). The best solution to this is to migrate to a different format such as CSV, where each row is not dependent on any other row in plain text. If someone would like to make a PR for this, I can review it. 

Alternatively, if someone wants to share their script to "fix" the slightly broken JSON code, that would work too. It should hopefully only be a simple matter of searching and replacing `][` strings with `],[` strings.

**Comment by vck3000** [comment]
Closed in 4f61cfecd44fc6ab03009cc88dd96b0d4cee5262.
